### DISCUSSION TITLE
¿Como actualizo un fork?

### DISCUSSION THREAD

**Discussion by venegas55** [LaunchX-InnovaccionVirtual/CursoIntroPython#209] [Q&A] [answered]
Como el título dice.... hasta anoche hice mis katas hasta la 6 pero veo que hay más....
tengo el fork de hace 1 semana pero quiero ver si hay forma de poder actualizar mi repo con esta misma... o tengo que "reforkear" y tener 2 repos con lo mismo?

**Comment by doguedogue** [comment, marked as answer]
Si lo tienes que sincronizar

https://docs.github.com/es/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork

**Comment by Dfredude** [comment]
Un compañero compartio el proceso detalladamente en esta discusion => #196

**Comment by DanielOe21** [comment]
Hola @venegas55 , tal como @Dfredude comentó, ya hay un post detallado sobre cómo actualizarlo:

👉 Clic aquí: https://github.com/LaunchX-InnovaccionVirtual/CursoIntroPython/discussions/196

Espero sea de ayuda, no olvides marcar tu pregunta como CONTESTADA si así lo consideras. Saludos!!

**Comment by charlyguz** [comment]
https://github.com/LaunchX-InnovaccionVirtual/CursoIntroPython/discussions/196

**Comment by venegas55** [comment]
Muchas gracias compañeros! lo reviso.

A terminar las katas nuevas hoy x.x

Exelente inicio de semana!
